script.js

```diff
@@ -32,8 +36,81 @@ const strokeWeight = 1.5;
 const pointRadius = 8;
 const fillOpacity = 0.7;
 
+let initialMapZoom = 4;
+//setting the initial zoom settings
+if (IS_MOBILE){
+  initialMapZoom = 3;}
+else if (IS_DESKTOP){
+  initialMapZoom = 5;}
+
+//initial values, if not given by the url
+let lat = 40.67;
+let long = -97.23;
+let z = initialMapZoom;
+let statesOn = true;
+let citiesOn = true;
+let countiesOn = true;
+
+//read url hash input
+let hash = location.hash;
+let input = inputValues(hash);
+
+// breaks up the url into an array; should be in the form
+// #/lat= & long= & z= & state= & cities= & counties=
```

Comment:
@goldfarb why the `#/lat` and not just `#lat`? Don't think the extra `/` is needed.

Reply:
I was matching the [crashmapper.org](http://crashmapper.org/) formatting

Reply:
@goldfarb the crashmapper.org site uses [URL search params](https://developer.mozilla.org/en-US/docs/Web/API/HTMLHyperlinkElementUtils/search), not the URL hash. I'd recommend on reading up on the difference between the two.